#inspiration from http://repository.timesys.com/buildsources/u/udev/udev-181/udev-181-define-trigger-happy.patch
#BTN_TRIGGER_HAPPY is not defined in /usr/include/linux/input.h for pre 2.6.34 kernels
# B. Hajgato April 30 2015
--- eudev-3.0/src/udev/udev-builtin-input_id.c.org	2015-04-30 09:11:07.990269206 +0200
+++ eudev-3.0/src/udev/udev-builtin-input_id.c	2015-04-30 09:12:19.407272116 +0200
@@ -33,6 +33,10 @@
 #include "udev.h"
 #include "util.h"
 
+#ifndef BTN_TRIGGER_HAPPY
+#define BTN_TRIGGER_HAPPY 0x2c0
+#endif
+
 /* we must use this kernel-compatible implementation */
 #define BITS_PER_LONG (sizeof(unsigned long) * 8)
 #define NBITS(x) ((((x)-1)/BITS_PER_LONG)+1)
